src/v2/guide/transitions.md

```diff
@@ -781,17 +781,17 @@ new Vue({
 </style>
 {% endraw %}
 
-Simultaneous entering and leaving transitions aren't always desirable though, so Vue offers some alternative **transition modes**:
+Les transitions simultanées d'entrée et de sortie ne sont pas toujours désirées, donc Vue propose des **modes de transition** alternatifs :
 
-- `in-out`: New element transitions in first, then when complete, the current element transitions out.
+- `in-out`: Les transitions du nouvel élément se passent en premier, puis quand elles sont terminées, les transitions de l'élément actuel s'effectuent.
```

Comment:
(*note: transitions n'est ici pas le nom pluriel mais la conjugaison du verbe to transition, "transitionner" en français. C'est un néologisme mais on peut se débrouiller pour ne pas l'utiliser*)

La transition entrante du nouvel élément s'effectue en premier, puis une fois terminée, déclenche la transition sortante de l'élément courant